airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/test/commands.py

```diff
@@ -1,19 +1,48 @@
 #
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
+from __future__ import annotations
 
 import logging
+from pathlib import Path
+from typing import TYPE_CHECKING
 
 import asyncclick as click
+import asyncer
 from pipelines.cli.click_decorators import click_ignore_unused_kwargs, click_merge_args_into_context_obj
 from pipelines.consts import DOCKER_VERSION
 from pipelines.helpers.utils import sh_dash_c
 from pipelines.models.contexts.click_pipeline_context import ClickPipelineContext, pass_pipeline_context
 
+if TYPE_CHECKING:
+    from typing import List, Tuple
+
+    import dagger
+
+## HELPERS
+async def run_poetry_command(container: dagger.Container, command: str) -> Tuple[str, str]:
+    """Run a poetry command in a container and return the stdout and stderr.
+
+    Args:
+        container (dagger.Container): The container to run the command in.
+        command (str): The command to run.
+
+    Returns:
+        Tuple[str, str]: The stdout and stderr of the command.
+    """
+    container = container.with_exec(["poetry", "run", *command.split(" ")])
+    return await container.stdout(), await container.stderr()
+
 
 @click.command()
 @click.argument("poetry_package_path")
-@click.option("--test-directory", default="tests", help="The directory containing the tests to run.")
+@click.option(
+    "-c",
+    "--poetry-run-command",
+    multiple=True,
```

Comment:
This means you can pass the option multiple times, not that you can pass multiple arguements to the option, right? An example in the docs of how to pass multiple may be helpful, as well as noting somewhere (new column?) that it can be passed multiple times.

Reply:
Yes you're right. I added an example to the README.